itou/users/models.py

```diff
@@ -493,6 +493,15 @@ def get_full_name(self):
         full_name = f"{self.first_name.strip().title()} {self.last_name.upper()}"
         return full_name.strip()[:70]
 
+    def get_truncated_full_name(self):
+        """
+        Return first name but display only last name's first letter for privacy.
+        """
+        name = self.first_name.strip().title()
+        if name and self.last_name:
+            name = f"{name} {self.last_name[0].upper()}"
```

Comment:
```suggestion
            name = f"{name} {self.last_name[0].upper()}."
```

?

Cf. https://www.notion.so/gip-inclusion/Inviter-un-collaborateur-devient-Ajouter-un-collaborateur-Modifier-les-textes-de-l-interface-et-18a5f321b604818c8083d4d64ef1b712?pvs=4#1b25f321b6048067835ec99bb12ac54d

Si c'est le cas, bien penser à retirer le point des templates.

Reply:
Effectivement ! Je modifie.